tests/functional/stn/xhr-runtime-patched.test.js

```diff
@@ -8,11 +8,18 @@ const testDriver = require('../../../tools/jil/index')
 let supported = testDriver.Matcher.withFeature('stn')
 
 testDriver.test('posts session traces when xhr is runtime patched', supported, function (t, browser, router) {
-  t.plan(2)
-
   let rumPromise = router.expectRum()
   let resourcePromise = router.expectResources()
-  let loadPromise = browser.get(router.assetURL('xhr-runtime-patched.html')).waitForFeature('loaded')
+  let loadPromise = browser.get(router.assetURL('instrumented.html', {
+    init: {
+      privacy: { cookies_enabled: false }
```

Comment:
I'm wondering instead of cookies_enabled false, if we should just clear the localStorage in the script string?  Nervous that we are no longer testing our prime case since cookies_enabled will be true for most customers